app/build.gradle.kts

```diff
@@ -50,16 +50,13 @@ android {
       path = file("src/main/cpp/CMakeLists.txt")
     }
   }
-  dexOptions {
-    javaMaxHeapSize = "4g"
```

Comment:
Is it not needed anymore?

Reply:
Yes, it's obsolete and not used anymore, 

```
WARNING:DSL element 'dexOptions' is obsolete and should be removed.
It will be removed in version 8.0 of the Android Gradle plugin.
Using it has no effect, and the AndroidGradle plugin optimizes dexing automatically.
```
is the message about it